AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -86,6 +86,7 @@ mvm.hu###footer-consent + #policy-accept-form
 telekom.hu##.cookie-policy-container
 vodafone.hu##div[data-js="_nudge, _cookiePolicy"].nudge-slim
 bonuszbrigad.hu##.cookieHeaderCenter
+montpellierhandball.com##.cookies-messages-wrapper
```

Comment:
I'm not sure, maybe they changed something, but on my end it's `id="cookies-messages-wrapper"`, not `class`.
Could you please check it again?
<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/29142494/52479018-80167900-2ba7-11e9-9b58-187f31b4a3c3.png)

</details>